code/src/index.css

```diff
@@ -1,13 +1,234 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: url('./assets/stock\ photos/stock-1.jpg') no-repeat center center fixed;
```

Comment:
The background image behaves really nice in the background when making it bigger or smaller!